projects/gloo/pkg/plugins/transformation/plugin.go

```diff
@@ -274,7 +274,8 @@ func (p *Plugin) ConvertTransformation(
 	// it will override the transformation-level settings
 	logRequestResponseInfo := stagedTransformations.GetLogRequestResponseInfo().GetValue()
 
-	if logRequestResponseInfo {
+	// if the global settings or the route/vhost settings are set to true, set all transformation-level settings to true
+	if p.logRequestResponseInfo || logRequestResponseInfo {
```

Comment:
I think the behavior we want with the child overrides (per the data plane pr) do we think its better to do the following?

```suggestion
	if stagedTransformations.GetLogRequestResponseInfo() == true ||  (stagedTransformations.GetLogRequestResponseInfo() == nil && p.logRequestResponseInfo)
```

Reply:
good call, thanks!